data/language/en-GB.txt

```diff
@@ -3642,6 +3642,8 @@ STR_6536    :This park was saved in a later version of OpenRCT2. The park was sa
 STR_6537    :Allow using regular paths as queue
 STR_6538    :Shows regular paths in the queues dropdown of the Footpaths window.
 STR_6539    :Brake Closed
+STR_6540    :{WINDOW_COLOUR_2}A Special Thanks to the following companies for their permission:
+STR_6541    :{WINDOW_COLOUR_2}Rocky Mtn. Construction, Wiegand
```

Comment:
Try to list the full name of the company if space permits; don't abbreviate it unless you need to to make it fit.
```suggestion
STR_6541    :{WINDOW_COLOUR_2}Rocky Mountain Construction, Wiegand
```

Reply:
I have mostly done it since RMC often abbreviates itself like it and it would take up a lot of space. I'll change it if others agree though.

Reply:
I agree with @James103 , company and product names typically have to retain their original name, otherwise you're changing their copyrighted/trademarked stuff.

I'd put here whatever they show on the copyright bottom section of their website, if there is any.

Reply:
Okay, how do you want me to list it then? As `Rocky Mountain Construction, Wiegand` or as `Rocky Mountain Construction Group, Josef Wiegand GmbH  Co. KG`?

Reply:
I'd go with the latter